### DISCUSSION TITLE
stereo pair with play5gen2 and five - can I do that with this project?

### DISCUSSION THREAD

**Discussion by bednim** [hklages/node-red-contrib-sonos-plus#255] [General]
Good day! This project is amazing! I have found out yesterday that you could stereo pair different speakers - is this correct? If so - can I pair play 5 gen2 with five speaker into stereo?

Thank you!

**Comment by hklages** [comment]
Good day. 

According to the [SONOS documentation](https://support.sonos.com/en-us/article/create-a-stereo-pair) both devices support stereo pairing - but it says also that you can only pair speakers of same type. 

The `household.create.stereopair` does not check the type - so you can try it. You can check the "result" with `household.get.groups` , property ChannelMap. 

There is another option to just group the two players. 

What provides better result (your opinion) - grouping or stereopair?

**Reply by bednim** [reply]
thank you very much for such detailed response.

when it comes to pairing - I prefer stereo - as then you are getting stereophonic audio :) 

Grouping is okay but I don't think it gives you best (or better?) experience.

Thank you once again.
